quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/ConfigCategory.java

```diff
@@ -4,13 +4,14 @@
     // ordered by name asc
     CLUSTERING("Cluster", 10),
     DATABASE("Database", 20),
-    FEATURE("Feature", 30),
-    HOSTNAME("Hostname", 40),
-    HTTP("HTTP/TLS", 50),
-    METRICS("Metrics", 60),
-    PROXY("Proxy", 70),
-    VAULT("Vault", 80),
-    LOGGING("Logging", 90),
+    TRANSACTION("Transactions",30),
```

Comment:
We don't have any other category in the plural, right? Isn't better `transaction`?